packages/host/tests/acceptance/commands-test.gts

```diff
@@ -1265,7 +1264,7 @@ module('Acceptance | Commands tests', function (hooks) {
     assert
       .dom('[data-test-boxel-alert="warning"]')
       .containsText(
-        'Command "show-card_566f" validation failed: data/attributes must NOT have additional properties',
+        'Command "show-card_566f" validation failed: data/attributes must have required property \'cardId\'',
```

Comment:
This seems like a sort of opposite error, did it change?

Reply:
Yes, I removed the restrictions about additional fields and added the ability to specify required fields.